test/e2e/antreapolicy_test.go

```diff
@@ -936,6 +938,134 @@ func testACNPDropEgress(t *testing.T, protocol v1.Protocol) {
 	executeTests(t, testCase)
 }
 
+// testACNPToServiceEndpoints tests that an ACNP is able to reject toService
+// traffic from pods labelled A to all Endpoints of the Service.
+func testACNPToServiceEndpoints(t *testing.T) {
+	svc1 := k8sUtils.BuildService("svc1", "x", 80, 80, map[string]string{"app": "c"}, nil)
```

Comment:
for completeness, could we have a test with a Service which has an empty selector? https://kubernetes.io/docs/concepts/services-networking/service/#services-without-selectors

Reply:
Added.